requirements.txt

```diff
@@ -1,9 +1,10 @@
 alembic==0.9.9
 blinker==1.4
 click==6.7
+faker
```

Comment:
We decided to use mimesis in that project. Try to eliminate that dependency and to replicate given usage with mimesis: https://lk-geimfari.github.io/mimesis/